PocketKit/Tests/SharedPocketKitTests/LegacyUserMigrationTests.swift

```diff
@@ -235,6 +235,8 @@ extension LegacyUserMigrationTests {
             return try! JSONSerialization.data(withJSONObject: correct)
         }
 
+        let expectation = XCTestExpectation(description: "Migration event fired successfully.")
+
```

Comment:
doesn't seem like we are using this expectation, curious on why its being added.